org.kde.labplot2.json

```diff
@@ -126,17 +123,20 @@
         },
         {
             "name": "kuserfeedback",
+            "config-opts": [
```

Comment:
Not sure why but the build log says

-- The following OPTIONAL packages have not been found:

for KUserFeedbackQt6

Reply:
If it's optional, I'm going to leave it out for this PR so we can get it merged. If we want it added, we can look into it in a subsequent change.